webapp/src/ts/services/get-summaries.service.ts

```diff
@@ -39,6 +39,8 @@ export class GetSummariesService {
     const reference =
       doc.patient_id ||
       (doc.fields && doc.fields.patient_id) ||
+      doc.patient_uuid ||
```

Comment:
As above, `doc.patient_uuid` should not be used. Only `doc.fields.patient_uuid` should be used.